packages/panna-sdk/src/react/consts/token-config.ts

```diff
@@ -18,7 +18,7 @@ export const liskTokenConfig: SupportedTokens = {
       icon: lskIcon
     },
     {
-      address: 'NATIVE_TOKEN_ADDRESS',
+      address: '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee',
```

Comment:
Please import `NATIVE_TOKEN_ADDRESS` from core and use that constant.
```suggestion
      address: NATIVE_TOKEN_ADDRESS,
```